src/Mod/Draft/draftguitools/gui_arcs.py

```diff
@@ -483,15 +482,10 @@ def __init__(self):
 
     def GetResources(self):
         """Set icon, menu and tooltip."""
-        _menu = "Arc by 3 points"
-        _tip = ("Creates a circular arc by picking 3 points.\n"
-                "CTRL to snap, SHIFT to constrain.")
-
-        d = {'Pixmap': "Draft_Arc_3Points",
-             'MenuText': QT_TRANSLATE_NOOP("Draft_Arc_3Points", _menu),
-             'ToolTip': QT_TRANSLATE_NOOP("Draft_Arc_3Points", _tip),
-             'Accel': 'A,T'}
-        return d
+        return {'Pixmap': "Draft_Arc_3Points",
+                'Accel': "A,T"}
+                'MenuText': QT_TRANSLATE_NOOP("Draft_Arc_3Points", "Arc by 3 points"),
+                'ToolTip': QT_TRANSLATE_NOOP("Draft_Arc_3Points", "Creates a circular arc by picking 3 points.\nCTRL to snap, SHIFT to constrain."),
```

Comment:
```suggestion
                'ToolTip': QT_TRANSLATE_NOOP("Draft_Arc_3Points", "Creates a circular arc by picking 3 points.\nCTRL to snap, SHIFT to constrain."),
                }
```

Reply:
Good catch. I should have tested... I'll fix this.